### ISSUE TITLE
Card not compatible with Frigate Beta ?

### ISSUE THREAD

**Issue by micAThome** [dermotduffy/frigate-hass-card#1323]
Regarding this issue here: https://github.com/dermotduffy/frigate-hass-card/issues/1317

I am using Frigate 0.12.1-367d724
As card I am using 5.2.0 (latest from here)

Tried today the latest **beta Frigate 0.13.0-beta6**.

The installation went smoothy, Frigate itself is running perfectly (better with this beta version).
But what is not "running", the card itself.
Getting the same error as the other user already reported:

`{
  "request": {
    "type": "frigate/events/get",
    "instance_id": "frigate",
    "cameras": [
      "Torkamera"
    ],
    "limit": 50,
    "id": 243
  },
  "response": {
    "code": "not_found",
    "message": "Unable to find Frigate instance with ID: frigate"
  }
}`

And before you answer the same thing:

> Guess: Issue not related to the card, but your Frigate integration is not working correctly. Check your Home Assistant logs, and/or reinstall the integration?

Frigate integration is running as it should (as the other user already said).
**Switching back to Frigate version 0.12.1 - voila, the card is working again.**

Conclusion: no, this issue has nothing todo with the Frigate integration (or maybe I missed something .. ?).
Maybe you try it by yourself, or just tell me what I can do to run the **Frigate beta version with a working frigate-hass-card**.

Thanks in advance.

**Comment by dermotduffy** [comment]
> Conclusion: no, this issue has nothing todo with the Frigate integration (or maybe I missed something .. ?).

This is a quick answer (on a flight): I would guess you are missing something (!), as that error message specifically means the _integration_ cannot find a Frigate instance with that `client_id` (not the card). Did you update your Frigate integration when you updated your Frigate version? You should be running [integration version v5.0.0-beta3](https://github.com/blakeblackshear/frigate-hass-integration/releases/tag/v5.0.0-beta.3).

I'd also bet (as per the error message), you'll  have something from the Frigate integration complaining in your Home Assistant logs ...

Note: I have not specifically tested the v5.2.0 card with the -beta6. If you are running the latest integration as above, and don't have anything relevant in your logs, please reply by pasting the diagnostics you get from holding down the Frigate menu button on the card.

Thanks and good luck.

**Comment by Mikescotland** [comment]
I have the same problem. Testing 0.13 beta 6 in docker, frigate integration latest beta.
No errors in frigate or HA.
However, I noticed one thing. All entities are fine except camera.XXX (mine are doorcam and outside). It looks like from some kind of reasons, Frigate integration proxy does not see them from frigate (red exclamation mark). But why?

**Comment by Mikescotland** [comment]
So. Main camera entity is unavailable. On this particular camera I've got detection of person or dog which creates two entities i.e. camera.doorcam_dog - these are having just a picture of the last recognised object.
When I choose them in the card configuration as a entity, not changing the go2rtc parameters which still point to the right stream "doorcam" - I've got the card working.
So essentially, the problem is with main camera entity unavailable.
No errors anywhere though...
